tests/test_validate_location.py

```diff
@@ -300,6 +300,45 @@ def test_location_is_fixed_when_there_is_tie_and_country_reference_in_ner_tags(
         assert response == expected_output
 
 
+class TestFilterNerCountries:
+    def test_filter_ner_tags_return_list_of_locations(self):
+        ner_tags = [
+            {"name": "Sidney", "label": "location"},
```

Comment:
Think you meant "Sydney"? I noticed you also did that [in other bits of the code](https://github.com/lifebit-ai/nlp-geocode-module/blob/master/tests/test_get_location.py#L151-L162) so may want to update those too?
```suggestion
            {"name": "Sydney", "label": "location"},
```